util/check-format.pl

```diff
@@ -537,7 +537,7 @@ sub check_nested_nonblock_indents {
   MATCH_COMMENT:
     if (my ($head, $opt_minus, $tail) = m|^(.*?)/\*(-?)(.*)$|) { # begin of comment: '/*'
         report("no SPC before '/*'")
-            if $head =~ m/[^\s\*]$/; # no space (nor '*', needed to allow '*/' here) before comment delimiter
+            if $head =~ m/[^\s(\*]$/; # no space (nor '(' or nor '*', needed to allow '*/' here) before comment delimiter
```

Comment:
The parenthesised part doesn't read well.  What about this:
`# not space, '(' or '*' (needed to allow '*/') before comment delimiter`

The same applies to the error report on the subsequent line.


As an grammatical aside: the wording _no x nor y_ is odd.  _Neither x nor y_ is typical.  Since this is introducing a third option I'd avoid the duality implications of _neither_ and use one of:

- _none of x, y, or z_
- _not x, y or z_

Reply:
Thanks Paul for your hints on using the word 'nor' in a grammatically correct way,
and how to replace it when there are more than two options.
Indeed my wording deteriorated here when adding the third option.
Your suggestion not only improves grammar but also the reference by the side note on '*/'.
I've followed it and improved also all other uses of 'nor'.